internal/website/gatherexamples/gatherexamples.go

```diff
@@ -14,22 +14,24 @@
 
 // Command gatherexamples extracts examples in a Go module into a JSON-formatted
 // object. This is used as input for building the Go CDK Hugo website.
-// Examples must include a comment that starts with "// This example is used in"
+//
+// Examples must include a comment
+// "// PRAGMA(gocloud.dev): Package this example for gocloud.dev."
 // somewhere in the function body in order to be included in this tool's output.
 //
 // gatherexamples does some minimal rewriting of the example source code for
 // presentation:
 //
 //   - Any imports the example uses will be prepended to the code.
 //   - log.Fatal(err) -> return err
-//   - A comment line "// Variables set up elsewhere:" will remove any code up
-//     to the next blank line. This is intended for compiler-mandated setup
-//     like `ctx := context.Background()`.
-//   - A comment line "// Ignore unused variables in example:" will remove any
-//     code until the end of the function. This is intended for
+//   - A comment line "// PRAGMA(gocloud.dev): Skip until next blank line." will
```

Comment:
s/skip/exclude/
s/skip/hide/

Reply:
Why not "skip"? They all seem about the same to me...?

Reply:
"Skip" was making me think of skipping compilation/execution or usage if a person was copying this example, not just hiding in display for a specific reuse of the example. I would even be more explicit "Hide lines to next blank in example" to avoid any confusion.

Reply:
Trying this:

`// PRAGMA(gocloud.dev): On gocloud.dev, hide lines until the next blank line.`

Reply:
Sounds good.

Are you opposed to my earlier suggestion to remove gocloud.dev in () in favor of havin git in the text and not both?
PRAGMA On gocloud.dev, hide the rest of the function
PRAGMA On gocloud.dev, add a blank import: _ "example.com/foo"
PRAGMA On gocloud.dev, hide lines until the next blank line

Reply:
Ah, OK; done. I left the colon though.